src/main/webapp/pipe.js

```diff
@@ -317,12 +322,14 @@ function isTaskLinkedToConsoleLog(data, task) {
 }
 
 function getPagination(showAvatars, component) {
-    var html = [];
-    if (!showAvatars && component.pagingData != '') {
-        html.push('<div class="pagination">');
-        html.push(component.pagingData);
-        html.push('</div>');
+    if (showAvatars || component.pagingData === '') {
```

Comment:
Here I have only inverted the condition to reduce nesting. This is common approach, which allows to make code more readable. Probably, will be used in future requests as well

Reply:
Readability is good!